Botkit/Microsoft.BotKit/BotWorkerConfiguration.cs

```diff
@@ -1,17 +1,35 @@
-using Microsoft.Bot.Builder;
-using Microsoft.Bot.Builder.Dialogs;
-using Microsoft.Bot.Schema;
-using System;
-using System.Collections.Generic;
-using System.Text;
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
 
 namespace Microsoft.BotKit
 {
+    using Microsoft.Bot.Builder;
```

Comment:
The same here with the using Statements.

Reply:
ACK

Reply:
Done